### PR TITLE
Create sum-bits.bril

### PR DESCRIPTION
**Pull request by stp59**
A small bril program which counts the number of ones in the bit-representation of a given integer.

### CODE REVIEW

File: docs/tools/bench.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ The current benchmarks are:
 * `sieve`: Print all prime numbers up to *n* using the [Sieve of Eratosthenes][sievee].
 * `collatz`: Print the [Collatz][collatz] sequence starting at *n*. Note: it is not known whether this will terminate for all *n*.
 * `ackermann`: Print the value of `Ack(m, n)`, the [two-argument Ackermann-Péter function][ackermann].
+* `sum-bit`: Print the number of bits in the binary representation of the input integer
```
Comment (sampsyo) [inline docs/tools/bench.md:19]:
```suggestion
* `sum-bit`: Print the number of bits in the binary representation of the input integer.
```

**Review by sampsyo — CHANGES_REQUESTED** [review]
Neat; looks cool! Please remember to add the test output files.

**Comment by stp59** [comment]
I think this should do it!

File: docs/tools/bench.md [hunk 2]
```diff
@@ -16,6 +16,7 @@ The current benchmarks are:
 * `sieve`: Print all prime numbers up to *n* using the [Sieve of Eratosthenes][sievee].
 * `collatz`: Print the [Collatz][collatz] sequence starting at *n*. Note: it is not known whether this will terminate for all *n*.
 * `ackermann`: Print the value of `Ack(m, n)`, the [two-argument Ackermann-Péter function][ackermann].
+* `sum-bit`: Print the number of bits in the binary representation of the input integer.
```
Comment (sampsyo) [inline docs/tools/bench.md:19]: Maybe this should be "the number of 1 bits"? It's not the totally length of the binary representation (i.e., the position of the highest 1).

**Review by sampsyo — COMMENTED** [review]
Looking good! One suggestion for the wording inside. Would you also mind resolving the conflicts? Thanks!!

**Comment by stp59** [comment]
Sorry I just now saw this. Conflicts resolved!

**Review by sampsyo — APPROVED** [review]
Great; thanks!

**Pull request merged**